docs/guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md

```diff
@@ -9,10 +9,230 @@ This guide will help you create a new Expectation Suite by profiling your data w
 <Prerequisites>
 
 - Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
+- Configured a [Datasource](../../tutorials/getting_started/connect_to_data.md)
 
 </Prerequisites>
 
-:::caution
-The UserConfigurableProfiler is not currently configured to work with the 0.13 Experimental API.
-We expect to update this in the near future.
+:::note
+The User Configurable Profiler makes it easier to produce a new Expectation Suite by building out a bunch of Expectations for your data.
+
+These Expectations are deliberately over-fitted on your data e.g. if your table has 10,000 rows, the profiler will produce an Expectation with the following config:
```

Comment:
When do we use backticks/code for GE-specific terms? Do we have a rule of thumb?

Reply:
Just spoke with Rachel about this and we want to reserve code blocks for instances where we're directly referencing a particular piece of code. Please disregard this!